package.json

```diff
@@ -29,7 +29,8 @@
     "proto3-json-serializer": "^1.0.0",
     "protobufjs": "7.2.2",
     "protobufjs-cli": "1.1.1",
-    "retry-request": "^5.0.0"
+    "retry-request": "^5.0.0",
+    "yargs": "^17.7.1"
```

Comment:
I'm looking forward to the split of google-gax into two modules :) For now, I'm hesitant of adding `yargs` as a runtime dependency. Even though it's not used by the code in runtime, it still adds a whole megabyte (!!!) to `node_modules`:

```
fenster-macbookpro:y fenster$ cat package.json 
{
  "dependencies": {
    "yargs": "^17.7.1"
  }
}
fenster-macbookpro:y fenster$ npm install 

added 16 packages, and audited 17 packages in 226ms

2 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
fenster-macbookpro:y fenster$ du -sh node_modules/
1.0M	node_modules/
```

which will increase not only the download size, but also a chance of us getting security vulnerability bugs. Since it's just about one extra command line option, would it be possible to parse `argv` manually? A simple `for` loop over `argv` should be enough here.